test/functional/wallet_taproot.py

```diff
@@ -172,9 +172,9 @@ class WalletTaprootTest(BitcoinTestFramework):
     """Test generation and spending of P2TR address outputs."""
 
     def set_test_params(self):
-        self.num_nodes = 2
+        self.num_nodes = 3
         self.setup_clean_chain = True
-        self.extra_args = [['-keypool=100'], ['-keypool=100']]
+        self.extra_args = [['-keypool=100'], ['-keypool=100'], ["-vbparams=taproot:1:1"]]
```

Comment:
nit: `self.nodes[1]` is not used in this test so can we keep 2 nodes and change `self.extra_args` for one?

Reply:
Node 1 will be used in #21365. I'd like to avoid conflicts with that as much as possible.